merge.go

```diff
@@ -112,6 +112,54 @@ func duplicateCreateNode(delta []Delta, msg string) bool {
 	return false
 }
 
+type EntityMergeAction struct {
+	Merge  *bool
+	Create *bool
+}
+
+func (e EntityMergeAction) GetMerge() bool {
+	if e == (EntityMergeAction{}) {
+		return true
+	}
+	if *e.Merge {
+		return true
+	}
+	return *e.Merge
+}
+
+func (e EntityMergeAction) GetCreate() bool {
+	if e == (EntityMergeAction{}) {
+		return true
+	}
+	if *e.Create {
+		return true
+	}
+	return *e.Create
+}
+
+func mergeSubtreeWithEntityAction(n, foundDBEntity *lpg.Node, nodeAssociations map[*lpg.Node]*lpg.Node, edgeAssociations map[*lpg.Edge]*lpg.Edge, dbGraph *DBGraph, nlayers []string, config Config) []Delta {
+	deltas := make([]Delta, 0)
+	for _, label := range nlayers {
```

Comment:
mergeActions:=EntityMergeActions{}
for _,label:=range nlayers {
    if op, ok:=config. ...; ok {
        mergeActions=op
        break
   }
}

// Move the if-else block here